app/assets/javascripts/components/alerts/campaign_alerts.jsx

```diff
@@ -19,11 +19,10 @@ class CampaignAlerts extends React.Component {
   }
 
   getCampaignSlug() {
-    return `${this.props.match.params.campaign_slug}`;
+    return `${this.props.campaign_slug}`;
```

Comment:
This change probably shouldn't include any changes to other tabs, since we have those working already... just changes isolated to the Home tab.

Reply:
It looks like git rebase command didn't fix all the changes. Now I fixed all those changes and pushed the code.